config/webpack.config.test.js

```diff
@@ -34,7 +34,12 @@ module.exports = function()
 					query: {
 						configFileName: './config/tsconfig.test.json'
 					}
-				}
+				},
+				{
+					test: /\.js$/,
```

Comment:
Yes for the test it's possible to import the JavaScript file with MediaQueries en DeviceState. Currently I commented out the JavaScript import from the test file. I did this because I actually didn't really know how to solve this. As the same code is tested for JavaScript I didn't want to create a whole new test file with the same testing methods only for testing the import. Any suggestions ?